core/admin/ExcAdminSite.py

```diff
@@ -0,0 +1,79 @@
+from django.contrib.admin import AdminSite
+from django.urls import NoReverseMatch, reverse
+from django.apps import apps
+from django.utils.text import capfirst
+
+
+class ExcursionAdmin(AdminSite):
+    index_template = "admin_index.html"
+
+    site_header = 'Excursion Admin'
```

Comment:
It is common practice to write constants like this: SITE_HEADER. This makes it easy to identify that something is (or should be) a constant while reading code

Reply:
This is set and used by the django backend, so it's not up to me